.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,19 @@
+<!--
+
+Template: QUESTION
+
+This is an issue for a feature request.
```

Comment:
Suggestion: "This is a template for a general question"

Reply:
Oh... sorry, 😓